docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -353,6 +358,7 @@ The JSON object is of the form:
 
 A PUT of the same format but with only the Meta fields will create the VisitLabel
 for this candidate, in an unstarted stage if the Visit label provided is valid.
+The SiteName is only required for users affiliated with multiple sites.
```

Comment:
Same issue, I don't think the API should vary based on the user using it.

Reply:
I agree. I will correct this.

Reply:
done